hatchet_sdk/clients/rest/api_client.py

```diff
@@ -450,13 +442,13 @@ def __deserialize(self, data, klass):
 
         if klass in self.PRIMITIVE_TYPES:
             return self.__deserialize_primitive(data, klass)
-        elif klass == object:
+        elif klass is object:
             return self.__deserialize_object(data)
-        elif klass == datetime.date:
+        elif klass is datetime.date:
             return self.__deserialize_date(data)
-        elif klass == datetime.datetime:
+        elif klass is datetime.datetime:
             return self.__deserialize_datetime(data)
-        elif klass == decimal.Decimal:
+        elif klass is decimal.Decimal:
```

Comment:
I'm not sure about this one - the linter flagged it, but would like eyes on this one!

Reply:
update: will probably revert this just to be safe